src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANDOM_COUNT = 100;
+
+    public Ball getRandomBall() {
+        return new Ball(new Random().nextInt(RANDOM_COUNT),new ColorSupplier().getRandomColor());

```

Comment:
Random and ColorSupplier should be a class fields. Also create separate  variable for random number and use it in constructor